src/main/java/org/kohsuke/github/GHMarketplaceAccount.java

```diff
@@ -72,4 +73,25 @@ public GHMarketplaceAccountType getType() {
         return type;
     }
 
+    /**
+     * Shows whether the user or organization account actively subscribes to a plan listed by the authenticated GitHub
+     * App. When someone submits a plan change that won't be processed until the end of their billing cycle, you will
+     * also see the upcoming pending change.
+     *
+     * <p>
+     * GitHub Apps must use a JWT to access this endpoint.
+     * <p>
+     * OAuth Apps must use basic authentication with their client ID and client secret to access this endpoint.
+     *
+     * @return a GHMarketplaceListAccountBuilder instance
+     * @throws IOException
+     *             in case of {@link IOException}
+     * @see <a href=
+     *      "https://docs.github.com/en/rest/apps/marketplace?apiVersion=2022-11-28#get-a-subscription-plan-for-an-account">Get
+     *      a subscription plan for an account</a>
+     */
+    public GHMarketplaceAccountPlan getPlan() throws IOException {
+        return new GHMarketplacePlanForAccountBuilder(root(), this.id).createRequest();
```

Comment:
@blacelle  Code coverage doesn't believe this line is covered by tests.  Please update to have it covered.  If it is easier to manually create data to cover this, that is fine too.  It just needs to be exercised.

Reply:
Latest commit should fix this. No additional data seemed require, supposedly because the underlying API call was done elsewhere.